packages/react-wildcat/src/server.js

```diff
@@ -107,6 +107,18 @@ function start() {
                 }));
             });
 
+            // Allow custom middleware priority over react/jspm/render from below
+            // otherwise server-only routes will receive 404's.
+            if (appServerSettings.middleware && appServerSettings.middleware.length) {
+                appServerSettings.middleware.forEach(function eachCustomAppMiddleware(middlewareConfigFunction) {
+                    if (typeof middlewareConfigFunction === "function" && middlewareConfigFunction.length >= 1) {
```

Comment:
Ah, then that might be too restrictive. I can see maybe passing in `wildcatConfig` along in a future update. Also if this check fails we would need to report to the user exactly their middleware is failing to initialize.